app/models/flood_risk_engine/organisation.rb

```diff
@@ -12,5 +12,8 @@ class Organisation < ActiveRecord::Base
       other: 5,
       unknown: 6
     }
+
+    belongs_to :primary_address, class_name: "FloodRiskEngine::Address"
```

Comment:
Ok given its the primary address I added migration so organisation had direct primary_address_id  

I'll remove migrations and change to has_one